### PR TITLE
Update target OS conditional for z/OS

### PR DESCRIPTION
**Pull request by danjapapajani**
Backport of https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/828

### CODE REVIEW

**Comment by keithc-ca** [comment]
Is this another back-port of https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/828?

**Comment by keithc-ca** [comment]
Please address the copyright check failure.

File: closed/make/modules/ibm.healthcenter/Copy.gmk [hunk 1]
```diff
@@ -32,7 +32,7 @@ $(HEALTHCENTER_EXTRACT) : $(HEALTHCENTER_JAR)
 	@$(RM) -rf $(@D)
 	$(call MakeDir, $(@D))
 	$(UNZIP) -q $< -d $(@D)
-  ifeq (zos,$(OPENJDK_TARGET_OS))
+  ifeq ($(call isTargetOs, zos), true)
```
Comment (keithc-ca) [inline closed/make/modules/ibm.healthcenter/Copy.gmk:35]: The copyright date needs to be updated in this file.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: closed/make/modules/ibm.healthcenter/Copy.gmk (hunk 1 above)
Comment (danjapapajani) [reply closed/make/modules/ibm.healthcenter/Copy.gmk:35]: It's been updated

Comment (keithc-ca) [reply closed/make/modules/ibm.healthcenter/Copy.gmk:35]: It has not; see line 2:
```
# (c) Copyright IBM Corp. 2021, 2021 All Rights Reserved
```

**Pull request merged**